features/TOCSidebar.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Table of Contents
+
+    @motivating
+    Scenario: Page table of contents displays headings in page
+        Given a user is viewing an article in the docs
+        Then they see a TOC on the right side of the page
+        And the TOC has heading "In this Article:"
+        And below the heading are a list of links from the article
+
+    @motivating
+    Scenario: Clicking on an item in TOC scrolls to linked heading in page
+        Given a user is viewing a docs article with TOC
+        When they click on a TOC link
+        Then the page scrolls to the corresponding section heading
+        And the TOC is still visible on the right side of the page
+
+    @ui
+    Scenario: Table of content items display hover state on hover
+        Given a user is viewing a docs article with TOC
+        When they hover over a link in TOC
+        Then the item's text is in bold
+        And the item's left border is green #007A5B
+
+    @motivating
+    Scenario: Introduction section appears on top of TOC
+        Given a user is viewing a docs article with TOC
+        Then the first link in the TOC is "Introduction"
+        When they click on "Introduction"
+        Then the page scrolls to the first heading
+
+    @motivating
+    Scenario Outline: Table of contents support nested links
+        Given a user is viewing a docs <article> with TOC
+        And the TOC has nested links
+        When they hover over a nested link
+        Then only the nested link is in the hover state
+        When they click the link
+        Then the page scrolls to the corresponding sub-section
+        Examples:
+            | article                                |
+            | /success/referral-program-optimization |
+            | /developer/segment                     |
+
+    @ui
+    Scenario Outline: Table of content scrolls when overflowing from view
+        Given a user is viewing a docs <article> with TOC
+        And the TOC is overflowing from view
+        Then they are able to scroll the TOC to access hidden links
+        Examples: 
+            | article                 |
+            | /designer/widget-editor |
+
+    @landmine
+    Scenario: Clicking on a link in TOC changes spacing in navigation bar
```

Comment:
I wonder if we can fix this quick, its likely actually unrelated to the TOC to a certain extent but might be worth trying to fix quick when we are in the code